tests/imposter/private-api.yaml

```diff
@@ -218,16 +218,16 @@ components:
               example: "2020/2021"
     Answer:
       required:
-        - "key"
-        - "value"
+        - "questionKey"
+        - "answer"
```

Comment:
Don't think we should be making changes here as this reflects the backend API which we don't control?